src/main/java/com/websafe/ccmsbe/controller/PrivacyRegulationController.java

```diff
@@ -0,0 +1,58 @@
+package com.websafe.ccmsbe.controller;
+
+
+import com.websafe.ccmsbe.entity.PrivacyRegulation;
+import com.websafe.ccmsbe.service.PrivacyRegulationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin("http://localhost:3000")
+@RequestMapping("api/v1/privacyRegulation")
```

Comment:
Let's follow the best practices in naming API endpoints. Refer to https://restfulapi.net/resource-naming/ to get an idea.

```suggestion
@RequestMapping("api/v1/privacy-regulations")
```